.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+sudo: false
+node_js:
+  - 6.11.1
```

Comment:
Interesting, is there a reason why our yml here uses a different node version (v6) than what we use locally (v8)?

Reply:
the docs for the node package say to use v6 https://www.notion.so/lob/Node-Package-a248ae47225548ebad0f727a4fe832a2

Reply:
Ah, could we update those docs? We'll want to make sure that our `.node-version` always matches the version we're using in CI, so let's go for 8.11.3 in this case!